dataset/transcribe.py

```diff
@@ -59,8 +61,9 @@ class DeepSpeech(TranscriptionModel):
     Credit: https://github.com/mozilla/DeepSpeech
     """
 
-    def __init__(self, model_path):
-        self.model = deepspeech.Model(model_path)
+    def __init__(self, language):
+        self.language=language
+        self.model = deepspeech.Model(os.path.join(paths["languages"], language, constants.TRANSCRIPTION_MODEL))
```

Comment:
I see what your doing here but this will break it for the CLI approach (the path's related stuff only applies to the GUI)